frontend/ui/screensaver.lua

```diff
@@ -383,6 +384,27 @@ function Screensaver:setMessage()
     self.input_dialog:onShowKeyboard()
 end
 
+function Screensaver:setStretchLimit(touchmenu_instance)
+    UIManager:show(SpinWidget:new{
+        value = G_reader_settings:readSetting("screensaver_stretch_limit_percentage", 8),
+        value_min = 0,
+        value_max = 25,
+        default_value = 8, -- percent
+        title_text = _("Set maximum stretch limit"),
+        ok_text = _("Set"),
```

Comment:
Maybe you need `ok_always_enabled = true` here?
If stretching was disabled and you want to enable it without changing the %.

Reply:
Indeed, thank you.